sdk/python/packages/flet-core/src/flet_core/video.py

```diff
@@ -559,36 +560,36 @@ def on_enter_fullscreen(self):
         return self._get_event_handler("enter_fullscreen")
 
     @on_enter_fullscreen.setter
-    def on_enter_fullscreen(self, handler: OptionalEventCallable):
+    def on_enter_fullscreen(self, handler: DefaultOptionalEventCallable):
         self._add_event_handler("enter_fullscreen", handler)
```

Comment:
I think instead of `DefaultOptionalEventCallable` (which sounds kind of heavy) we could use just `EventCallable` everywhere. We don't have non-optional event handlers, right?